plugins/testPlugin/testPlugin.cpp

```diff
@@ -422,11 +429,12 @@ void mumble_onKeyEvent(uint32_t keyCode, bool wasPress) {
 
 bool mumble_hasUpdate() {
 	// This plugin never has an update
-	return false;
+	return true;
 }
 
 MumbleStringWrapper mumble_getUpdateDownloadURL() {
-	static const char *url = "https://i.dont.exist/testplugin.zip";
+	static const char *url = "https://github.com/hbeni/fgcom-mumble/releases/download/v0.4.0/fgcom-mumble-linux-0.4.0.tar.gz";
+	//static const char *url = "https://i.dont.exist/testplugin.zip";
```

Comment:
Did this accidentally sneak in?
:)

Reply:
Yes indeed. That's a remnant of my local update tests ^^